openedx/core/djangoapps/oauth_dispatch/adapters/dot.py

```diff
@@ -76,3 +79,20 @@ def get_token_scope_names(self, token):
         Given an access token object, return its scopes.
         """
         return list(token.scopes)
+
+    def is_client_restricted(self, client_id):
+        """
+        Returns true if the client is set up as a RestrictedApplication.
+        """
+        application = self.get_client(client_id=client_id)
+        return RestrictedApplication.objects.filter(application=application).exists()
+
+    def get_authorization_filters(self, client_id):
+        """
+        Get the authorization filters for the given client application.
+        """
+        application = self.get_client(client_id=client_id)
+        filters = [org_relation.to_jwt_filter_claim() for org_relation in application.organizations.all()]
+        if application.authorization_grant_type == application.GRANT_CLIENT_CREDENTIALS:
```

Comment:
This needs to change from an equality to an inequality.  See a commit I just pushed with the updated doc.